src/porepy/models/solution_strategy.py

```diff
@@ -727,6 +727,15 @@ def update_time_dependent_ad_arrays(self) -> None:
         """
         self.update_all_boundary_conditions()
 
+    def darcy_flux_keywords(self) -> list[str]:
+        """Return the keywords for which the Darcy flux discretization is stored.
```

Comment:
It is not discretization, but values, right?

Reply:
You're right!